script/helperFunctions.sh

```
@@ -1480,23 +1480,28 @@ function verifyContract() {
-  while [ $COMMAND_STATUS -ne 0 -a $RETRY_COUNT -lt $MAX_RETRIES ]; do
-    # execute verification command
-    eval "$VERIFY_CMD"
-    COMMAND_STATUS=$?
 
-    # increase retry counter
-    RETRY_COUNT=$((RETRY_COUNT + 1))
+  # skip this part if API key is SOURCIFY_API_KEY
+  if [[ "$API_KEY" != "SOURCIFY_API_KEY" ]]; then
+    # Attempt verification with retries
+    echo "Trying to verify contract $CONTRACT on $NETWORK with address $ADDRESS using command: $VERIFY_CMD"
+    while [ $COMMAND_STATUS -ne 0 -a $RETRY_COUNT -lt $MAX_RETRIES ]; do
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)